examples/datasets_converters/convert_inshop.py

```diff
@@ -80,7 +80,6 @@ def build_inshop_df(dataset_root: Path, no_bboxes: bool) -> pd.DataFrame:
 
         df = df.merge(df_bbox, on="image_name", how="inner")
         df.reset_index(inplace=True, drop=True)
-        df["path"] = df["image_name"].apply(lambda x: Path(dataset_root) / x)
```

Comment:
The logic of `path` construction differs. I don't know the details, but is it ok that in the new version of the code we always do `x.replace("img/", "img_highres/")`?